scalableminds/webknossos
# Make users confirm the deletion of the initial task node

From https://discuss.webknossos.org/t/protect-the-initial-node-from-being-deleted/539/

> I think we should do this:
>  show a message that needs to be confirmed if the user tries to delete the node with the task ID1 (I guess this is always the ID if we create tasks with given coordinates?)
> the tracing needs to be part of a task (no explorative please because exploratives are only done by scientists or very experienced hiwis).
> Suggested message: "Do you really want to delete the initial node?"
> 
> if the user tries to delete a tree that contains the initial node there should ideally be a different tree-deletion warning: This tree contains the initial node. Do you really want to delete the whole tree?

### Mailable description of changes:
 - The deletion of the initial node of a task (or of the tree that contains that initial node) now needs to be confirmed explicitly by the user.

### Steps to test:
- Open a task and try to delete the initial node or tree. Confirms should be shown in each case.
- Delete other nodes, no confirm should be shown.
- Delete a tree that is not the initial tree, a different confirm should be shown.
- Nothing should have changed in explorative tracings.

### Issues:
- fixes #2090

------
